src/main/java/utils/DataPropLoader.java

```diff
@@ -19,4 +19,8 @@ public static String loadProperties(final String resource, final String key) {
     public static String getBrowserProperty(String keyProp) {
         return loadProperties("browser.properties", keyProp);
     }
+
+    public static String getNodeCapabilityProperty(String keyProp) {
```

Comment:
It might be easier to get what this properties about just naming it like grid.properties